glib-macros/src/properties.rs

```diff
@@ -398,11 +398,10 @@ fn expand_properties_fn(props: &[PropDesc]) -> TokenStream2 {
     quote!(
         fn derived_properties() -> &'static [#crate_ident::ParamSpec] {
```

Comment:
>  Full removal will break a contract of glib::Properties macro.

That's OK though :) Can you update it accordingly?

Reply:
Done

Reply:
What contract does that break though? That change is completely internal or not?

Reply:
Users of the macro who use `once_cell` will need to change their properties to std cell too.

Reply:
That is fine